src/psyclone/tests/psyGen_test.py

```diff
@@ -70,6 +71,14 @@
 GOCEAN_BASE_PATH = os.path.join(os.path.dirname(os.path.abspath(__file__)),
                                 "test_files", "gocean1p0")
 
+
+# Module fixtures
+
+@pytest.fixture(scope="module")
+def f2008_parser():
```

Comment:
Please add a docstring describing this fixture.

Reply:
Done